src/curl.cpp

```diff
@@ -3563,28 +3563,30 @@ int S3fsCurl::CompleteMultipartPostRequest(const char* tpath, const std::string&
     }
 
     // make contents
-    std::string postContent;
-    postContent += "<CompleteMultipartUpload>\n";
+    std::string* pPostContent = new std::string();
```

Comment:
I don't think this is right -- it just more complicated so cppcheck cannot see the dangling pointer.  See #1622 for an alternate approach.

Reply:
Thank you, the error handling of dnglingLifetime was incorrect.
I have corrected it, so please reconfirm it.